frontend/src/pages/home/Home.jsx

```diff
@@ -102,13 +109,45 @@ const Home = () => {
                   />
                 ))}
               </Space>
-            </Row>
+            </Row> */}
           </Col>
         </Row>
       </div>
       <div className="home-even highlights">
+        <div className="body">
+          <Row justify="space-between">
+            <Col lg={10}>
+              <div className="report-wrapper">
+                <div className="description">
+                  <h1>Real Time Management Information System</h1>
+                  <p>
+                    The Kenya Rural Urban Sanitation and Hygiene (RUSH) platform
+                    is a real-time monitoring and information system owned by
+                    the Ministry of Health. The platform aggregates quantitative
+                    and qualitative data from county and national levels and
+                    facilitates data analysis, report generation and
+                    visualizations.
+                  </p>
+                </div>
+                <ul>
+                  <li>
+                    <FiCheckCircle />
+                    <span>Report generation and visualizations.</span>
+                  </li>
+                </ul>
+                <Button type="primary">{text.welcomeCta}</Button>
+              </div>
+            </Col>
+            <Col lg={8}>
```

Comment:
10 + 8 is 18 @ifirmawan, please fix it to 24

Reply:
but please add padding to the image, so it doesn't touched the paragraph on the left